src/index.ts

```diff
@@ -34,7 +35,19 @@ mongoose.connect(config.mongoose.url!).then(() => {
     // // Parse json request body
     app.use(express.json());
 
-    // // sanitize request data
+    // Get image from s3 by key
+    app.get('/images/:key', async (req: Request, res: any) => {
```

Comment:
This works but it doesn't reduce the load on your service.
It would be nice to pull images from S3 directly on your front end. 
You can either make the bucket public or keep it non-public and make the uploaded images public.